src/client/Game/SelectOptions/styles.css

```diff
@@ -20,6 +20,13 @@
   width: 280px;
 }
 
+.gameSelectOptionsFlavorImage {
+  display: block;
+  margin: 0px auto;
+  width: 90%;
+}
+
+
```

Comment:
nit: extra line

Reply:
Oh, the eyes of a true coder! Marcus! Able to suss out and feel the imbalance created by a stone in the cogs of beauty.  A tragic extra line breaking the rhythm of the CSS. The gentle ebb and flow of the CSS.  Both serene and tempestuous.  An open brace followed so delicately, so carefully by its twin. A mirror. A symmetry. Such beauty. Such benevolence.  Thank you. Marcus, you are data itself.

Reply:
Done.